src/main/ml-modules/root/lib/facetsLib.mjs

```diff
@@ -328,66 +336,13 @@ function _getViaSearchFacet(
   };
 }
 
-function _getViaSearchFacetValues(
-  facetName,
-  facetConfig,
-  baseSearchCriteria,
-  filterResults,
-  requestOptions
-) {
-  // Log mining script matches on this message.
-  xdmp.trace(
-    traceName,
-    `Searching for the '${facetName}' facet values (filterResults: ${filterResults}).`
-  );
-  const page = 1;
-  // always search for the MAXIMUM_SEMANTIC_PAGE_LENGTH + 1, this way we can return up to MAXIMUM_SEMANTIC_PAGE_LENGTH, and also know if there are more facet values than the max
-  const pageLength = MAXIMUM_SEMANTIC_PAGE_LENGTH + 1;
-  const facetValues = search({
-    searchCriteria: facetConfig.getFacetValuesCriteria(baseSearchCriteria),
-    page,
-    pageLength,
-    filterResults,

```

Comment:
I removed the associated log mining script code: `viaSearchFacet` was fairly prevalent in it.